src/mailing.py

```diff
@@ -25,7 +24,7 @@ def json_to_excel(json_data: dict) -> str:
     :param json_data:
     :return:
     """
-    file_pth = os.path.join(os.path.dirname(__file__), "tmp", datetime.now().strftime("%Y-%m-%d_%H-%M-%S")+".xlsx")
+    file_pth = os.path.join(os.path.dirname(__file__), "tmp", datetime.now().strftime("%Y-%m-%d_%H-%M-%S") + ".xlsx")
```

Comment:
White space is not recommended in functions' arguments.
https://peps.python.org/pep-0008/#other-recommendations